Index: src/View/Actors/AuthenticationVIew.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/View/Actors/AuthenticationVIew.java b/src/View/Actors/AuthenticationView.java
rename from src/View/Actors/AuthenticationVIew.java
rename to src/View/Actors/AuthenticationView.java
--- a/src/View/Actors/AuthenticationVIew.java	
+++ b/src/View/Actors/AuthenticationView.java	
@@ -4,9 +4,9 @@
 
 import java.util.Scanner;
 
-public class AuthenticationVIew {
+public class AuthenticationView {
 
-    public AuthenticationVIew(){}
+    public AuthenticationView(){}
 
     public User login(){
         Scanner scanner = new Scanner(System.in);
